src/storage.cc

```diff
@@ -276,6 +297,12 @@ Status Storage::RestoreFromBackup() {
     LOG(ERROR) << "[storage] Failed to reopen db: " << s2.Msg();
     return Status(Status::DBOpenErr, s2.Msg());
   }
+
+  // Clean up backup engine
+  backup_->PurgeOldBackups(0);
```

Comment:
the backup_ is not being  initialized under the new code

Reply:
In line 284, we open it
that just is compatible with full synchronization with old version kvrocks. Compatibility makes me headache 😭